### ISSUE TITLE
ABNF > Blab - Core rules not exported

### ISSUE THREAD

**Issue by stestagg** [katef/kgt#58]
When generating blab output from ABNF that uses built-in core rules, the blab output
doesn't have those core rules defined, so blab can't parse the fule:

```
$> echo "a = DIGIT" | kgt -l abnf -e blab | blab
Could not find /usr/share/blab/DIGIT.blab needed for 'DIGIT'
```

**Comment by katef** [comment]
This is true, and it's actually intentional - for other dialect outputs, I didn't want to provide definitions for these rules where we might already have them from another context. Especially for fuzzing with blab, I often want to define these by hand to something specific, instead of using their full definitions.

What I think we should have though, is something in examples/ (it needn't be written in ABNF!) specifying them. Then you could render them both to blab and cat them together.
